docs/99-references/00-references.adoc

```diff
@@ -21,34 +21,56 @@ This section contains references that are cited in the curriculum.
 
 **B**
 
-- [[[bass,Bass+ 2012]]] Len Bass, Paul Clements, Rick Kazman: Software Architecture in Practice. 3rd Edition, Addison-Wesley, 2012.
+- [[[bass,Bass+ 2021]]] Len Bass, Paul Clements, Rick Kazman: Software Architecture in Practice. 4rd Edition, Addison-Wesley, 2021.
 
+- [[[bellotti,Bellotti 2021]]] Marianne Bellotti: Kill It with Fire: Manage Aging Computer Systems (and Future Proof Modern Ones), No Starch Press, 2021.
 
-**C**
 
-- [[[clements,Clements+ 2001]]] Paul Clements, Rick Kazman, Mark Klein: Evaluating Software Architectures Methods and Case Studies. Addison-Wesley, 2001.
+**D**
+
+- [[[demeyer,Demeyer+ 2023]]] Serge Demeyer, Stéphane Ducasse and Oscar Nierstrasz: Object-Oriented Reengineering Patterns, 2003.
```

Comment:
2023 oder 2003? Hinten die Jahreszahl sagt 2003, die Referenz 2023.

Reply:
it should be 2003.